lib/publiccloud/provider.pm

```diff
@@ -618,6 +619,7 @@ sub terraform_apply {
     foreach my $i (0 .. $#{$vms}) {
         my $instance = publiccloud::instance->new(
             public_ip => @{$ips}[$i],
+            resource_group => $resource_group,
```

Comment:
but resource group is relevant only for Azure and this code is used by all 3 providers :thinking: